dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatRequestSettings.cs

```diff
@@ -42,6 +42,13 @@ public class ChatRequestSettings
     /// </summary>
     public IList<string> StopSequences { get; set; } = Array.Empty<string>();
 
+    /// <summary>
+    /// How many completions to generate for each prompt. Default is 1.
+    /// Note: Because this parameter generates many completions, it can quickly consume your token quota.
+    /// Use carefully and ensure that you have reasonable settings for max_tokens and stop.
+    /// </summary>
+    public int ResultsPerPrompt { get; set; } = 1;
```

Comment:
Azure SDK dropped the ball on this one. In my opinion this should just be `N`. It's what all the core APIs reference / document

Reply:
Not sure what you mean here, but in the docs we make no mention to any upper limit.

Reply:
Ah I was referring purely to the naming. Aka `N` instead of `ResultsPerPrompt` naming